src/session/session.ts

```diff
@@ -76,6 +76,17 @@ export default class Session {
     this.data[keys.CLIENT_SIG] = data.digest("hex");
   }
 
+  public appendData(key: string, value: any): void {
+    this.data[key] = value;
+  }
+
+  public accessToken(): string | undefined {
```

Comment:
I can't see where this method is called. Is it needed (right now)?

Reply:
called from SessionService